### PR TITLE
Feature/iphonex

### PR DESCRIPTION
**Pull request by Melizzo**
# PR Description
Please include a summary of the change:
Media Queries for Spirited Quartini app

Commit Emojis:
:tada: Initial commit
:art: CSS modifications/updates.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
Ability to view via iphone x

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Tests (non-breaking change which adds testing)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/CocktailDetails/CocktailDetails.scss [hunk 1]
```diff
@@ -57,4 +57,11 @@
 
 .details-img {
 	width: 55%;
+}
+
+@media only screen and (max-width: 375px) {
```
Comment (asodemann18) [inline src/CocktailDetails/CocktailDetails.scss:62]: makes sense

File: src/Header/Header.scss [hunk 2]
```diff
@@ -25,63 +25,133 @@ header {
...
-	font-family: 'Recursive', sans-serif;
-	min-height: 21px;
-	margin-bottom: 4px;
+  cursor: pointer;
+  font-family: "Recursive", sans-serif;
+  min-height: 21px;
+  margin-bottom: 4px;
+}
+
+@media only screen and (max-width: 375px) {
```
Comment (asodemann18) [inline src/Header/Header.scss:88]: thanks for doing this!

**Comment by LeahWinters** [comment]
Great job! Thanks for pairing with me on this part so I could understand how to use media queries!

File: src/Login/Login.scss [hunk 3]
```diff
@@ -74,4 +74,31 @@ displayNone {
 	min-height: 26px;
 	padding-left: 3%;
 	width: 250px;
+}
+
+@media only screen and (max-width: 375px) {
```
Comment (asodemann18) [inline src/Login/Login.scss:79]: nice!

**Review by asodemann18 — APPROVED** [review]
Great work!

**Pull request merged**